modules/ipi-install-network-requirements.adoc

```diff
@@ -155,4 +149,4 @@ You can reconfigure the control plane nodes to act as NTP servers on disconnecte
 [id='network-requirements-out-of-band_{context}']
 == Port access for the out-of-band management IP address
 
-The out-of-band management IP address is on a separate network from the node. To ensure that the out-of-band management can communicate with the `baremetal` node during installation, the out-of-band management IP address address must be granted access to the TCP 6180 port.
+The out-of-band management IP address is on a separate network from the node. To ensure that the out-of-band management can communicate with the `baremetal` node during installation, the out-of-band management IP address address must be granted access to the TCP 6180 port.
```

Comment:
I am also unsure about the "TCP 6180 port" I looked at the ISG and saw a small section(Web, IP, and email addresses p.226 Definitions section) that states:

> Use specific punctuation to separate the elements of a web address: insert a colon and two forward slashes after the abbreviated protocol name, insert a colon before a port number, and insert a forward slash before a directory name or file name.

I am not sure if that applies in this case. I will have someone verify me here.